<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>520</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>759</width>
    <height>520</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>759</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="main_label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>10</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Böl</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_select_file">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lütfen dosyanızı seçiniz:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_select">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>60</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>PDF Seç</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>611</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_aralk">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>150</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Aralık giriniz:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>170</y>
     <width>351</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Birden çok aralık için virgül kullanın. (örn. 15-25, 40-43)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>200</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>401</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_onizle">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>99</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Önizle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kaydet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_divide">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Böl</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>370</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="placeholder">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>331</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gotham Narrow</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>Önizleme için sağ alttaki butonu aktif hale getiriniz.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
